aiida_quantumespresso/utils/validation/trajectory.py

```diff
@@ -30,9 +31,6 @@ def verify_convergence_trajectory(trajectory, index=-1, threshold_forces=None, t
     else:
         converged_stress = True
 
-    if converged_forces is None or converged_stress is None:
-        return None
-
     return converged_forces & converged_stress
```

Comment:
Any reason to use bitwise-and here, not `and`? They have the same behavior for booleans, but e.g.

```python
In [5]: True & 1
Out[5]: 1

In [6]: True & 2
Out[6]: 0

In [7]: True and 2
Out[7]: 2
```

Reply:
Note that that was already there and have to say I don't remember any specific reason. So happy to change it to normal boolean logic. Were there any other things you wanted to change in this PR? If not then I will make that change and then we can merge. I have two other more important PRs with bugs blocked by this

Reply:
Yeah, just noticed it due to the adjacent change - mostly I was wondering if the `converged_*` variables might be some nontrivial type, but looking at the `verify_*` functions they're just bools. So, doesn't matter either way, although `and` is slightly more readable.

Reply:
Have made the change